src/main/kotlin/me/zeroeightsix/kami/feature/module/Aura.kt

```diff
@@ -120,11 +125,17 @@ object Aura : Module() {
         return false
     }
 
+    private fun checkWeapon(stack: ItemStack): Boolean {
+        return (stack.item is AxeItem || stack.item is SwordItem)
+    }
+
     private fun attack(player: ClientPlayerEntity, e: Entity, onlyUse32k: Boolean) {
         var holding32k = false
+        var holdingWeapon = false
```

Comment:
i don't see a reason to assign this here and then reassign it later. maybe you could remove this variable entirely and write the check directly into the `if`